osu.Game/Rulesets/UI/DrawableRulesetDependencies.cs

```diff
@@ -119,33 +119,34 @@ public FallbackSampleStore(ISampleStore primary, ISampleStore fallback)
 
             public Stream GetStream(string name) => primary.GetStream(name) ?? fallback.GetStream(name);
 
-            public IEnumerable<string> GetAvailableResources() => throw new NotSupportedException();
+            public IEnumerable<string> GetAvailableResources() => primary.GetAvailableResources() ?? fallback.GetAvailableResources();
```

Comment:
`NotSupportedException` does not mean "todo". these methods should never be called by anyone, because nobody should expect them to return anything sane.